project_script/datatreatment.py

```diff
@@ -19,3 +21,35 @@ def is_bigger_data_frame(data_frame_1, data_frame_2):
 
 def equalize_data_frame_rows(data_frame_1, data_frame_2):
     return data_frame_1.loc[0:len(data_frame_2)-1, :]
+
+
+def smooth(array):
+    temp_list = []
+    for key, values in enumerate(array):
+        temp_list.append((savgol_filter(array[key], 55, 2).tolist()))
+    return temp_list
+
+
+def find_the_peaks(data_as_list):
```

Comment:
Suggest: you could just type the function name like `find_peaks()`, as the same you did in the line 40. `find_peak_index`, and not `find_the_peak_index`.